### ISSUE TITLE
Add a new step (optional step)

### ISSUE THREAD

**Issue by Omar8345** [mszeles/selenideium-element-inspector#30]
Hi there, I installed the extension/plugin **today**.

I tried **testing the extension**, I got an error as I use **Microsoft Edge** and actually _it automatically blocks trackers_. Even I understood what I need to do to fix after thinking, but I think noting that will make it easier for developers who are **stuck**.

What to do?

Add a **note/(optional) step** to unblock trackers **(or unblock the script)** so the plugin would work **properly as expected**

Nice note

**I will be making the changes to save work on you :)**

_Also I am sad because I cannot make a Stripe account to withdraw my **Buymeacoffee.com reward**, can you have my reward in any other way?_ 😊

**Comment by mszeles** [comment]
Hi @Omar8345 !

Thanks for your feedback. 

I will be honest, I do not really understand what you are trying to say here. 

You say you are using Microsoft Edge but Selenideium Element Inspector is a Chrome plugin so it was never planned to work in Edge.

Can you please tell me the exact reproduction steps in Chrome?

Thanks,
Miki

**Comment by Omar8345** [comment]
Hi @mszeles!

**Microsoft Edge now allows to download extensions on MS Edge from the Chrome Web Store, so you can expect to get users using your plugin on MS Edge**... As **MS Edge** blocks trackers by default, it is as it blocks the plugin from running, so the solution is either to unblock **ALL** trackers, or **the script** so then the **plugin would work properly**

**Comment by mszeles** [comment]
Hi @Omar8345!

Thanks for the report. Selenideium Element Inspector only supports Chrome. I do not have the time to support and test this extension on Microsoft Edge.

If somebody would like to save 1 hour daily, then installing Chrome is not a huge price for it.

I am closing this issue as won't do, but if you would like to have Microsoft Edge support, you can submit a feature request.

I really appreciate your enthusiasm so I will reward you with one coffee for this request, one coffee for the pull request, and also one coffee if you submit the Edge support request.

I will clarify the details of the Selenideium Element Inspector Bounty Hunting Reward Program, and I will also add them to the readme.

As you cannot get a buymeacoffee account I am open to any other solution (Revolut maybe) where I can send you the 3*$5.

**Comment by Omar8345** [comment]
Hi, in total I get 3 coffees now, 15$ total, this issue, the PR, and link issue, can you give my reward in PayPal?
